data/json/recipes/other/cords_and_ropes.json

```diff
@@ -81,6 +162,22 @@
     "proficiencies": [ { "proficiency": "prof_fibers_rope", "required": false, "time_multiplier": 1.5 } ],
     "components": [ [ [ "rope_6", 5 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "rope_30",
+	"id_suffix":"spinwheel_ropes",
+	"batch_time_factors": [ 70, 1 ],
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "tailor",
+    "time": "1 m",
```

Comment:
I think these times are a bit short. Getting the fibers onto the wheel alone would probably take a minute or two.